src/Feed.php

```diff
@@ -51,7 +57,7 @@ public function toResponse($request): Response
 
     protected function resolveItems($resolver): Collection
     {
-        $resolver = array_wrap($resolver);
+        $resolver = Arr::wrap($resolver);
```

Comment:
`array_wrap()` is deprecated and will be remove in 5.9 so I swapped this out to the static version.